src/js/video.js

```diff
@@ -162,6 +162,11 @@ function videojs(id, options, ready) {
 
   options = options || {};
 
+  // Store a copy of the el before modification, if it is to be restored in destroy()
+  if (options.restoreEl === true) {
+    options.restoreEl = el.cloneNode(true);
```

Comment:
to get this line covered by tests, we probably want to add a test to https://github.com/videojs/video.js/blob/main/test/unit/video.test.js

Ideally, we'd have a test per embed type https://videojs.com/guides/embeds/

Reply:
I'll add some more tests. And sine this can be a minor I'll rebase onto `main`. Before starting I was thinking this would be more impactful a change than it is.